### DISCUSSION TITLE
第3节课笔记(20班)

### DISCUSSION THREAD

**Discussion by Farewell-CK** [InternLM/Tutorial#342] [笔记]
宝子们，20班的笔记交这里哦

**Comment by TLzh** [comment]
https://blog.csdn.net/zhiho14/article/details/135567637

**Comment by deyouzuishuai** [comment]
https://zhuanlan.zhihu.com/p/678101900

**Comment by liu673** [comment]
https://github.com/liu673/InternLM_OpenNotebook/blob/master/lesson_3/lesson_3_note.md

**Comment by LiyanJin** [comment]
https://blog.csdn.net/weixin_53443275/article/details/135686219

**Comment by 20300240011** [comment]
https://zhuanlan.zhihu.com/p/679656869
